### PR TITLE
Text change after job confirmation

### PR DESCRIPTION
**Pull request by swetha571**
# Type of change? 

- [x] Bug fix
- [ ] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 
Text change after job confirmation.

...

**Checklist:**

- [ ] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: src/components/flyout/deviceReconfigureFlyout.js [hunk 1]
```diff
@@ -306,20 +306,19 @@ class DeviceReconfigureFlyout extends React.Component {
...
-                onClick={this.applyDeviceConfigureJobsData}
-              /> }
-        </div>
+        <SummarySection count={totalAffectedDevices} content={this.state.jobApplied ? lang.DEVICES_RECONFIGURED : lang.AFFECTED_DEVICES} />
+        {this.state.jobApplied ? null :
+          <div className="btn-container">
+            <PcsBtn svg={CancelX} value={lang.CANCEL} onClick={this.props.onClose} />
+            {this.state.showSpinner && <Spinner size="medium" />}
+            <PcsBtn
+                  className="primary"
```
Comment (spryor) [inline src/components/flyout/deviceReconfigureFlyout.js]: indentation is off here

File: src/components/flyout/deviceScheduleFlyout.js [hunk 2]
```diff
@@ -163,19 +163,17 @@ class DeviceScheduleFlyout extends React.Component {
...
-                      onClick={this.onConfirm}
-                      disabled={disabledButton}>{lang.APPLY}</PcsBtn>
-                }
-              </div>
+              {this.state.jobApplied ? null :
+                <div className="btn-group">
+                  <PcsBtn svg={CancelX} onClick={this.props.onClose}>{lang.CANCEL}</PcsBtn>
+                  {this.state.showSpinner && <Spinner size="medium" />}
+                  <PcsBtn svg={Apply}
+                        className="primary"
```
Comment (spryor) [inline src/components/flyout/deviceScheduleFlyout.js]: indentation is off here

File: src/components/flyout/deviceScheduleFlyout.js [hunk 3]
```diff
@@ -163,19 +163,17 @@ class DeviceScheduleFlyout extends React.Component {
...
-                {this.state.showSpinner && <Spinner size="medium" />}
-                {this.state.jobApplied
-                  ? <PcsBtn svg={Apply} disabled>{lang.APPLIED}</PcsBtn>
-                  : <PcsBtn svg={Apply}
-                      className="primary"
-                      onClick={this.onConfirm}
-                      disabled={disabledButton}>{lang.APPLY}</PcsBtn>
-                }
-              </div>
+              {this.state.jobApplied ? null :
```
Comment (spryor) [inline src/components/flyout/deviceScheduleFlyout.js]: `!this.state.jobApplied &&`

File: src/components/flyout/deviceTagFlyout.js [hunk 4]
```diff
@@ -420,19 +420,18 @@ class DeviceTagFlyout extends React.Component {
...
-            ? <PcsBtn svg={Apply} value={lang.APPLIED} disabled />
-            : <PcsBtn
-                className="primary"
-                svg={Apply}
-                value={lang.APPLY}
-                disabled={disabledButton}
-                onClick={this.applyDeviceTagJobsData}/> }
-        </div>
+        <SummarySection count={totalAffectedDevices} content={this.state.jobApplied ? lang.DEVICES_TAGGED : lang.AFFECTED_DEVICES} />
+        {this.state.jobApplied ? null :
```
Comment (spryor) [inline src/components/flyout/deviceTagFlyout.js]: `!this.state.jobApplied &&`

File: src/common/lang.js [hunk 5]
```diff
@@ -79,6 +79,9 @@ const lang = {
   DEVICES: 'Devices',
   DEVICES_ARE_PROVISIONING: 'devices are provisioning...',
   DEVICES_TO_PROVISION: 'devices to provision',
+  DEVICES_TAGGED: 'devices tagged',
+  DEVICE_SCHEDULED: 'devices scheduled',
+  DEVICES_RECONFIGURED: 'device reconfigured',
```
Comment (dluc) [inline src/common/lang.js]: should this be "devices" (plural)?

File: src/common/lang.js [hunk 6]
```diff
@@ -79,6 +79,9 @@ const lang = {
   DEVICES: 'Devices',
   DEVICES_ARE_PROVISIONING: 'devices are provisioning...',
   DEVICES_TO_PROVISION: 'devices to provision',
+  DEVICES_TAGGED: 'devices tagged',
+  DEVICE_SCHEDULED: 'devices scheduled',
```
Comment (dluc) [inline src/common/lang.js]: should the constant name use plural like the other constants?

File: src/components/flyout/deviceReconfigureFlyout.js [hunk 7]
```diff
@@ -306,20 +306,19 @@ class DeviceReconfigureFlyout extends React.Component {
...
-            : <PcsBtn
-                className="primary"
-                svg={Apply}
-                value={lang.APPLY}
-                disabled={disabledButton}
-                onClick={this.applyDeviceConfigureJobsData}
-              /> }
-        </div>
+        <SummarySection count={totalAffectedDevices} content={this.state.jobApplied ? lang.DEVICES_RECONFIGURED : lang.AFFECTED_DEVICES} />
+        {!this.state.jobApplied &&
```
Comment (spryor) [inline src/components/flyout/deviceReconfigureFlyout.js]:
```
{
    ... multi line code
}
```

File: src/components/flyout/deviceTagFlyout.js [hunk 8]
```diff
@@ -420,19 +420,18 @@ class DeviceTagFlyout extends React.Component {
...
-            ? <PcsBtn svg={Apply} value={lang.APPLIED} disabled />
-            : <PcsBtn
-                className="primary"
-                svg={Apply}
-                value={lang.APPLY}
-                disabled={disabledButton}
-                onClick={this.applyDeviceTagJobsData}/> }
-        </div>
+        <SummarySection count={totalAffectedDevices} content={this.state.jobApplied ? lang.DEVICES_TAGGED : lang.AFFECTED_DEVICES} />
+        {!this.state.jobApplied &&
```
Comment (spryor) [inline src/components/flyout/deviceTagFlyout.js]: multi line `{}` styling

File: src/components/flyout/deviceScheduleFlyout.js [hunk 9]
```diff
@@ -163,19 +163,19 @@ class DeviceScheduleFlyout extends React.Component {
...
-                {this.state.showSpinner && <Spinner size="medium" />}
-                {this.state.jobApplied
-                  ? <PcsBtn svg={Apply} disabled>{lang.APPLIED}</PcsBtn>
-                  : <PcsBtn svg={Apply}
-                      className="primary"
-                      onClick={this.onConfirm}
-                      disabled={disabledButton}>{lang.APPLY}</PcsBtn>
-                }
-              </div>
+              {!this.state.jobApplied &&
```
Comment (spryor) [inline src/components/flyout/deviceScheduleFlyout.js]:
```
{
    ... multi line code
}
```

**Pull request merged**